index.html

```diff
@@ -0,0 +1,248 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Dunk card display</title>
+		<link rel="stylesheet" type="text/css" href="quize.css">
+	</head>
+	<body>
+
+		<article>
+			<header>
+				<h1>2017 Nissan 370Z Roadster</h1>
+			</header>
+
+			<section class="section1">
+
+				<img src="http://lln.mnmcdn.com/photos/cars/nissan/big-images/2013-nissan-370z-roadster-2_653.jpg">
+
+				<p class="discription">
+					The Nissan 370Z Roadster is a rear-wheel drive convertible that blends athletic proportions with track-tuned handling. The convertible is available in two trim levels, and buyers who prefer the comfort of a closed roof can opt for the coupe model.
+				</p>
+
+				<p class="lable"> Available </p>
+
+			</section>
+
+			<section class="section2">
+
+				<header>
+				  <h2>specifications</h2>
+				</header>
+
+				  <div>
```

Comment:
I feel like you put these extra divs around your p tags in order to target them in the css. The same could be accomplished by giving them a class. (The requirement called for multiple block level children. The div only counts as one, which in turn has its own block level elements.)